src/components/views/sul-study-place/filtering-table/study-place-filtering-table.tsx

```diff
@@ -237,7 +237,7 @@ const StudyPlaceFilteringTable = ({items}: Props) => {
                 <Td className="min-w-1/5 block w-auto sm:border-b sm:border-black-40 md:text-left lg:table-cell lg:w-1/5 lg:pr-32">
                   <Link
                     href={item.branchPath}
-                    className="text-16 font-normal leading-[23px] transition-colors hover:bg-black-10 hover:text-brick-dark hover:no-underline focus:bg-none focus:text-cardinal-red active:text-cardinal-red"
+                    className="inline-block text-16 font-normal leading-[23px] underline transition-colors hover:bg-black-10 hover:text-brick-dark hover:no-underline focus:bg-none focus:text-cardinal-red active:text-cardinal-red"
```

Comment:
Do you need the `inline-block`?

Reply:
Yes, that gives the background color they want.

Reply:
Can you use `block` instead? I am not a fan of anything `inline`

Reply:
The reason is that `block` fills the whole `td` instead of hugging the content like they want. 
![Screenshot 2024-08-12 at 11 06 11 AM](https://github.com/user-attachments/assets/5917ab33-c9af-4a41-876c-4771237940d6)

Reply:
`w-fit` would solve that.

Reply:
Ok. That worked.  Thanks!